src/login/logind-dbus.c

```diff
@@ -2001,7 +2001,7 @@ static int verify_shutdown_creds(
 
         if (blocked) {
                 /* We don't check polkit for root here, because you can't be more privileged than root */
-                if (uid == 0 && (flags & SD_LOGIND_ROOT_CHECK_INHIBITORS))
```

Comment:
Also update the document to state that this flag is implied/has no effect on newer logind?